asapdiscovery-docking/asapdiscovery/docking/scripts/run_docking_oe.py

```diff
@@ -389,7 +389,7 @@ def main():
 
     # load ml models
     if args.gat:
-        GAT_model = GATInference("gat_test_v0")
+        GAT_model = GATInference("asapdiscovery-GAT-2023.04.12")
```

Comment:
we should probably print/log the specific model string, handy for debugging.

Reply:
(even though the version will be retrievable from the .yaml, during debugging we'll stumble on the logged version before thinking of checking the .yaml)

Reply:
@JenkeScheen should be fixed now. :)